components/dashboard/donationCard.js

```diff
@@ -58,15 +58,17 @@ const DonationCard = ({ donationAmount, donationLoading, hasDonation, optOutOfAd
                 outlineColor: 'currentColor !important'
               }}
               aria-label='Edit monthly donation'
-              icon='edit'
+              icon='pencil'
               title='Edit monthly donation'
             />
           </Flex>
         )}
         <UnderlinedHeading
           as='h3'
           id='donation-amt'
-          text='Monthly Donation'
+          text='Donated monthly'
```

Comment:
i dont think donated monthly implies what the card shows. the card shows an active donation, nothing past tense about it, let's revert the text change

Reply:
pete and i worked on wording in his PR - keep those changes